build.gradle

```diff
@@ -1,36 +1,17 @@
-task wrapper(type: Wrapper) {
-    gradleVersion = '2.6'
-}
```

Comment:
https://github.com/VTCSecureLLC/ace-android/commit/7ba298789fd51e72ffb1f8c319ad2a2c9a2eb5db 
It was part of the  crashlythicsndk commit